python/lsst/daf/butler/core/dbAuth.py

```diff
@@ -212,5 +212,47 @@ def getAuth(self, drivername, username, host, port, database):
 
         raise DbAuthError(
             "No matching DbAuth configuration for: "
-            "({}, {}, {}, {}, {})".format(
-                drivername, username, host, port, database))
+            f"({drivername}, {username}, {host}, {port}, {database})")
+
+    def getUrl(self, url):
+        """Fill in a username and password in a database connection URL.
+
+        This function parses the URL and calls `getAuth`.
+
+        Parameters
+        ----------
+        url : `str`
+            Database connection URL.
+
+        Returns
+        -------
+        url : `str`
+            Database connection URL with username and password.
+
+        Raises
+        ------
+        DbAuthError
+            Raised if the input is missing elements, an authorization
+            dictionary is missing elements, the authorization file is
+            misconfigured, or no matching authorization is found.
+
+        See also
+        --------
+        `getAuth`
```

Comment:
Somehow sphinx does not like this line:
```
Exception occurred:
  File "/home/salnikov/.local/lib/python3.7/site-packages/numpydoc/docscrape.py", line 255, in parse_item_name
    raise ParseError("%s is not a item name" % text)
numpydoc.docscrape.ParseError: `getAuth` is not a item name in 'Fill in a username and password in a database connection URL.\n\nThis function parses the URL and calls `getAuth`.\n\nParameters\n----------\nurl : `str`\n    Database connection URL.\n\nReturns\n-------\nurl : `str`\n    Database connection URL with username and password.\n\nRaises\n------\nDbAuthError\n    Raised if the input is missing elements, an authorization\n    dictionary is missing elements, the authorization file is\n    misconfigured, or no matching authorization is found.\n\nSee also\n--------\n`getAuth`'
The full traceback has been saved in /tmp/sphinx-err-djoayeel.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```
It stops crashing if I change this line to something like

    Method `getAuth`

but then there is not `getAuth` link in the output.

Reply:
Apparently backticks should not be used in `see also` section, stripping backticks stops crashing and link is generated OK.